docs/NETWORK-MODELS.md

```diff
@@ -0,0 +1,87 @@
+# Network Models
+
+What are the network models that EVE supports for Edge Container workloads, and how are they implemented?
+
+The first half of this document describes _what_ those network models are and how they impact Edge Container communications. The second half describes _how_ those models are implemented.
+
+## Models
+
+Fundamentally, each Edge Container may need to communicate with services outside itself. Those can be one or more of:
+
```

Comment:
Small nit: not sure that 'Those can be one or more of'... no networking at all reads that well.

Reply:
No problem. Emphasized the "may" and removed the "no networking at all".